### PR TITLE
fix: fixed maxQueryLimit validation

### PR DESCRIPTION
**Pull request by WillieRuemmele**
@W-8768271@

### CODE REVIEW

File: test/unit/config/configTest.ts [hunk 1]
```diff
@@ -146,6 +146,36 @@ describe('Config', () => {
           expect(err).to.have.property('name', 'InvalidConfigValue');
         }
       });
+      describe('maxQueryLimit', () => {
+        it('will throw an error', async () => {
```
Comment (shetzel) [inline test/unit/config/configTest.ts:150]:
```suggestion
        it('will throw an error when the value is mixed alphanumeric', async () => {
```

File: test/unit/config/configTest.ts [hunk 2]
```diff
@@ -146,6 +146,36 @@ describe('Config', () => {
...
+          const config: Config = await Config.create(Config.getDefaultOptions(true));
+          try {
+            config.set('maxQueryLimit', '123abc');
+            assert.fail('Expected an error to be thrown.');
+          } catch (err) {
+            expect(err).to.have.property('name', 'InvalidConfigValue');
+          }
+        });
+        it('will throw an error', async () => {
```
Comment (shetzel) [inline test/unit/config/configTest.ts:159]:
```suggestion
        it('will throw an error when the value is not a number', async () => {
```

File: test/unit/config/configTest.ts [hunk 3]
```diff
@@ -146,6 +146,36 @@ describe('Config', () => {
...
+          const config: Config = await Config.create(Config.getDefaultOptions(true));
+          try {
+            config.set('maxQueryLimit', 'abc');
+            assert.fail('Expected an error to be thrown.');
+          } catch (err) {
+            expect(err).to.have.property('name', 'InvalidConfigValue');
+          }
+        });
+        it('will set config value', async () => {
```
Comment (shetzel) [inline test/unit/config/configTest.ts:168]:
```suggestion
        it('will set config value when the value is a stringified number', async () => {
```

File: test/unit/config/configTest.ts [hunk 4]
```diff
@@ -146,6 +146,36 @@ describe('Config', () => {
...
+          const config: Config = await Config.create(Config.getDefaultOptions(true));
+          const res = config.set('maxQueryLimit', '123');
+          expect(res.maxQueryLimit).to.equal('123');
+        });
+        it('will set config value', async () => {
```
Comment (shetzel) [inline test/unit/config/configTest.ts:173]:
```suggestion
        it('will set config value when the value is a number', async () => {
```

**Review by shetzel — CHANGES_REQUESTED** [review]
Nits, but will make the test output more explicit and self-documenting

**Review by shetzel — APPROVED** [review]

**Review by peternhale — APPROVED** [review]

**Pull request merged**